### DISCUSSION TITLE
android 6.1 m3s how to update install bootstrap in termuxV0.72

### DISCUSSION THREAD

**Discussion by 00000000002** [termux/termux-app#3547] [Q&A] [answered]
android 6.1 how to update install bootstrap in termuxV0.720 
 SHOW ERROR MEG：![IMG_20230816_054326.jpg](https://github.com/termux/termux-app/assets/52356585/f25df918-4e79-45cb-bcf0-41080da84593)

**Comment by CallMeCaleb94** [comment]
Are you kidding? Take your phone off airplane

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6
